.github/test-a-feature.md

```diff
@@ -103,4 +103,49 @@ yarn test
+
+### Tests with Steps API
+
+This default test only checks the rendering for the component in its initial state. In order to test the rendering of more complex components, such as a `Dropdown`, screener provides an [api](https://www.npmjs.com/package/screener-runner) to execute actions on the DOM, in a way similar to end-to-end tests. These tests are automatically sent to Screener in our test runs, as long as the tests and their files respect the conventions:
+- the test file should be placed at the same location as the tested component in the docs.
+- the test file should be named exactly as the component file. If `DropdownExample.shorthand.tsx` is to be tested, the screener test file should be `DropdownExample.shorthand.steps.ts`.
+- the tests should be written as an array of callbacks that accept a `steps` parameter, as they will be chained in `screener.config` automatically. The `steps` parameter is actually the `Steps` object from screener, instantiated in `screener.config`.
+
+#### Example for a test file:
+
+```tsx
+import { Dropdown } from '@stardust-ui/react'
+
+const steps = [
+  steps => steps.click(`.${Dropdown.slotClassNames.triggerButton}`)
+    .snapshot('Opens with selected item highlighted'),
+  steps => steps.hover(`.${Dropdown.slotClassNames.itemsList} li:nth-child(2)`)
+    .snapshot('Highlights another item'),
+]
+
+export default steps
+```
+
+#### Important mentions:
+
+- by convention, each test is written as a different callback and added to the steps array.
+- an actual assertion is performed by taking a `.snapshot(<Your test name here>)`, as the assertion is performed by comparing the screenshot with the one taken on the previous run and considered correct.
+- a test can have as many snapshots as needed.
+- before the snapshot is taken, steps are added to reach the state of assertion, with methods from `screener-runner` API (`.click(<selector>)`, `.hover(<selector>)`etc.)
+- tests do not perform any cleanup by default. This means a test is dependent on the state of the component in which the previous test leaves it. In the example above, the second test will receive the `Dropdown` component with the list open.
+
+### Run Screener tests
+In order to run the tests locally, make sure to have a Screener API key saved in the environment variables on your machine. For instance, on MacOS/Linux you can use `export SCREENER_API_KEY=<Your screener key here>`.
+
+When ran locally, not all the tests may be needed to run, only the ones added/edited. This can be achieved by changing the regexp in `screener.config.js`, where the `states` property is created. Make sure not to commit that change though! Results can be viewed in the Screener online dashboard, among all the other runs (for master, pull requests, etc.)
```

Comment:
For future work, we may allow user to send the regexp when running the command :)

Reply:
added https://github.com/stardust-ui/react/issues/860